.github/workflows/release.yml

```diff
@@ -1,15 +1,12 @@
-name: CI
+name: 'GH Actions - Release'
```

Comment:
Here too, let's match the `name` value to the filename.

Reply:
updated `gh-actions-release`